update expected timeline for public disclosure (#193)
kubernetes/committee-security-response: comms-templates/distributors-announcement-email.md

### EMBARGOED

> The information contained in this email is **[under embargo](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy)** until the scheduled public disclosure is sent on **$DATE. The exact time may be at OR after 9AM PT**.

Reviewer: ```suggestion
The information contained in this email is **[under embargo](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy)** until the public disclosure is sent. The disclosure is scheduled to be sent on $DATE at or after 9AM PT.
```
